### ISSUE TITLE
Support converting line endings when adding text file to image

### ISSUE THREAD

**Issue by gray-space** [AppleCommander/AppleCommander#27]
Could AppleCommander automatically fix line endings on text files being added to an image? 

When I used the command:

```bash
ac -p micoldev.po source/TESTDHR txt < testhdr.txt 
```
on Linux, the files comes through with Linux line endings ($0A, aka newlines) rather than Apple II line endings ($0D). 

Granted, I can get around the issue by using:
```bash
cat TESTDHR.txt | tr '\n' '\r' | ac -p micoldev.po source/TESTDHR txt 
```
but that is cumbersome.

**Comment by a2geek** [comment]
Sounds like a good idea!

**Comment by gray-space** [comment]
@a2geek: that should work. The issue I was hitting mainly as that the line endings were causing the Micol Basic editor to not display the source code I had transferred. It seemed fine with the high-bit not being set, once the line ending was fixed. 

Looking at some of the code I wrote natively on the  Apple II in the hex dump, it does have the high-bit set, and the separator is in those files $8D.
